app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt

```diff
@@ -241,6 +263,7 @@ class NotificationRepository(
 
     companion object {
         private const val DEVICE_UUID = "prefs.string.device_uuid"
+        private const val REGISTRATION_UPDATE_FAILED = "prefs.boolean.registration_update_failed"
```

Comment:
Should this variable have a KEY_ prefix?

Reply:
sure